.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,34 @@
+name: Bug Report
+description: Report any bugs you encounter.
+body:
+  - type: textarea
+    attributes:
+      label: What is the issue?
+      description: Insert a detailed description of what your issue is
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: What is the issue with?
+      options:
+        - Deployment
+        - Effis
```

Comment:
effis is still a part of eludris, I'd recommend removing the `Eludris` option and adding ones for the rest of the microservices + todel